catalogs/catalog-common/build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
```

Comment:
I would suggest using a different package name, common will probably be used later on. you can think of a different name like "bundled-catalog" for example.